### PR TITLE
added support for java.util.logging (JUL)

### PR DESCRIPTION
**Pull request by raupachz**
In july we started using logentries for one of our products. We are using java util loggings (JUL) instead of log4j or logback, so we could not use your official framework. I came up with a custom handler that supports JUL. Its running in production right know and we are quite happy with the result.

This handler might reach a greater audience in your framework than in mine. Its currently hosted in raupachz/jul--ext.

### CODE REVIEW

**Comment by samjamesobrien** [comment]
Just to let you know this will be assessed soon, we are changing some of our internal versioning and deployment methods for public libraries.

**Comment by raupachz** [comment]
No problem. I think I need to adjust the pull request, too. The buffer size is too small.

**Comment by samjamesobrien** [comment]
I am hesitant to accept this pull request as it removes support for openjdk6, do you have an alternative implementation that doesn't use StandardSocketOptions, introduced in 1.7?

https://travis-ci.org/logentries/le_java/jobs/48481965

We want to maintain maximum compatibility with older jdks for library code.

**Comment by raupachz** [comment]
I don't think SO_KEEPALIVE is needed. If the connections drops the LogentriesHandler is trying to reconnect. I removed the corresponding line. I haven't tried this in production yet.

**Comment by raupachz** [comment]
Hmm, I messed up with rebase. Shouldn't have done that.

**Comment by samjamesobrien** [comment]
Are you still working on your rebase issue or are you happy with the current state of the pull request?

I won't evaluate the code if it is still being fixed, thanks!

**Comment by raupachz** [comment]
The pull request is fine. The git history is just messed up. I don't care unless you do.

**Comment by samjamesobrien** [comment]
I'll evaluate it in the coming days, thanks!

File: src/test/resources/logging.properties [hunk 1]
```diff
@@ -0,0 +1,10 @@
+handlers = com.logentries.jul.LogentriesHandler
+
+.handlers = com.logentries.jul.LogentriesHandler
+
+com.logentries.jul.LogentriesHandler.token = f2ff093a-54c7-4305-9449-1dccda748347
```
Comment (samjamesobrien) [inline src/test/resources/logging.properties]: Maybe this should be aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee?

Comment (raupachz) [inline src/test/resources/logging.properties]: Allright. No problem here. This was just the token I used for testing.

Comment (samjamesobrien) [inline src/test/resources/logging.properties]: As long as it isn't a real token :) You could end up with a lot of unwanted logs.

Comment (raupachz) [inline src/test/resources/logging.properties]: Of course it is not a real token. That would be stupid. ;-)

**Pull request merged**